Gruntfile.js

```diff
@@ -95,14 +104,147 @@ module.exports = function(grunt) {
 				base: DOC_DEST
 			},
 			src: ['**']
+		},
+
+		'export': {
+			target: {
+				src: ['src/sketch/*.sketch']
+			},
+			options: {
+				platforms: ['ANDROID', 'IOS', 'WEB'],
+			},
 		}
 
 	});
 
 
 	grunt.registerTask('optimize', ['svgmin']);
+  grunt.registerTask('export', ['export']); //'clean:icons',
 	grunt.registerTask('dist', ['optimize', 'svgstore']);
 
-	grunt.registerTask('default', ['clean', 'dist', 'preprocess']);
+	grunt.registerTask('default', ['clean:all', 'dist', 'preprocess']);
 	grunt.registerTask('ghpages', ['default', 'gh-pages']);
+
+	grunt.registerTask('export_artboards', 'export artboards', function(artboardNames, src, platform) {
+		var done = this.async();
+		var platformOptions = {};
+		var isValidPlatform = true;
+		var platformName = platform.toUpperCase();
+		var destination;
+
+		switch(platformName){
+			case 'WEB':
+				destination = DIST_ICONS_WEB
+				platformOptions = {
+					formats: 'svg',
+					scales: '1.0'
+				}
+				break;
+			case 'ANDROID':
+				destination = DIST_ICONS_ANDROID
+				platformOptions = { // REAL ANDROID OPTIONS TBD
+					formats: 'png',
+					scales: '1.0'
+				}
+				break;
+			case 'IOS':
+				destination = DIST_ICONS_IOS
+				platformOptions = {
+					formats: 'pdf',
+					scales: '1.0'
+				}
+				break;
+
+			default:
+				console.log('platform is invalid')
+		}
+
+		grunt.util.spawn({
+			cmd: 'sketchtool',
+			args: [
+				'export',
+				'artboards',
+				src,
+				'--items=' + artboardNames,
+				'--scales=' + platformOptions.scales,
+				'--output=' + destination,
+				'--formats=' + platformOptions.formats
+				]
+		}, function(error, result, code){
+			done();
+		});
+	});
+
+	var getArtboards = function(artboardJSON, array, platformName) {
```

Comment:
It would be good to doc the params on some of these functions. For `getArtboards`, we should return a new array instead of mutating the `array` param. It would also be good to name `array` something more descriptive